wp-content/themes/core/blocks/hero/hero.php

```diff
@@ -1,5 +1,5 @@
 <?php
 declare( strict_types=1 );
-
-$model = new \Tribe\Project\Blocks\Types\Hero\Model( $args['block'] );// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
```

Comment:
Can you add [a PHPDoc above this](http://p.tri.be/EtRbLP) so that PHPStorm doesn't highlight `$args` as undefined: http://p.tri.be/uMJSIq?